web/src/pages/orders/[orderId]/edit/index.page.jsx

```diff
@@ -0,0 +1,112 @@
+import { Button } from 'flowbite-react'
+import { FaTasks } from 'react-icons/fa'
+
+import DatePicker from '@/components/organisms/DatePicker'
+import FilePickerInput from '@/components/organisms/FilePickerInput '
+import PageHeader from '@/components/organisms/PageHeader'
+import SelectInput from '@/components/organisms/SelectInput'
+import TextAreaInput from '@/components/organisms/TextAreaInput'
+import TextInput from '@/components/organisms/TextInput'
+import Template from '@/components/templates/Template'
+import { branchOptions, statusOptions } from '@/hooks/const'
+
+import { useHooks } from './hooks'
+
+const Order = ({ orderId }) => {
+  const { formState, handleSubmit, categories } = useHooks(orderId)
```

Comment:
orderId its not being passed somewhere, and not in use in hooks
```suggestion
const Order = () => {
  const { formState, handleSubmit, categories } = useHooks()
```